FeedStoreChallenge/CoreData Feed Store/CoreDataFeedStore.swift

```
@@ -0,0 +1,127 @@
+//
+//  CoreDataFeedStore.swift
+//  FeedStoreChallenge
+//
+//  Created by Nicolas De Maio on 9/20/20.
+//  Copyright © 2020 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+public class CoreDataFeedStore: FeedStore {
+    private let container: NSPersistentContainer?
+    private let context: NSManagedObjectContext?
```

This class cannot work without an `NSPersistentContainer` or an `NSManagedObjectContext`, so those dependencies shouldn't be optional. If it fails to create them, you could throw an error in the initializer to prevent the class from being created in an invalid state.